api/tests/core/offerers/test_api.py

```diff
@@ -2505,3 +2517,5 @@ def test_do_not_recreate_new_user_offerer_if_invitation_already_accepted(self):
         user_offerers = offerers_models.UserOfferer.query.all()
 
         assert len(user_offerers) == 2
+        actions_list = history_models.ActionHistory.query.all()
+        assert len(actions_list) == 0
```

Comment:
Suggestion, plus simplement : 
```suggestion
        actions_list = history_models.ActionHistory.query.count() == 0
```